extensions/solidtime/src/views/shared/CrudActions.tsx

```diff
@@ -0,0 +1,59 @@
+import { Action, ActionPanel, Icon } from "@raycast/api";
+import type { Fn } from "../../utils/types.js";
+
+type CrudActionsProps = {
+  name: string;
+  onDetails?: Fn;
+  onDelete?: Fn;
+  onEdit?: Fn;
+  onNew?: Fn;
+
+  /** Additional Actions */
+  itemActions?: React.ReactNode;
+  generalActions?: React.ReactNode;
+};
+export function CrudActions(props: CrudActionsProps) {
+  return (
+    <ActionPanel>
+      <ActionPanel.Section>
+        {props.onDetails && (
+          <Action
+            title="Show Details"
+            icon={Icon.Eye}
+            // shortcut={{ modifiers: [], key: "enter" }}
+            onAction={props.onDetails}
+          />
+        )}
+        {props.onEdit && (
+          <Action
+            title={`Edit ${props.name}`}
+            icon={Icon.Pencil}
+            shortcut={{ modifiers: ["cmd"], key: "e" }}
+            onAction={props.onEdit}
+          />
+        )}
+        {props.itemActions}
+        {props.onDelete && (
+          <Action
+            title={`Delete ${props.name}`}
+            icon={Icon.Trash}
+            shortcut={{ modifiers: ["cmd"], key: "d" }}
```

Comment:
Could we use `Keyboard.Shortcut.Common.Remove` from `@raycast/api`

Reply:
And should we show a `confirmAlert` too?

Reply:
Amazing, did not know there are predefined shortcuts, will include them for sure!
The confirm delete probably makes sense, will include
Thanks!